### PR TITLE
Send age and gender to GA

### PR DESCRIPTION
**Pull request by erikh360**

### CODE REVIEW

File: iogt/middleware.py [hunk 1]
```diff
@@ -61,3 +73,48 @@ def request_is_secure(self, request):
...
+        path = request.get_full_path()
+        referer = request.META.get('HTTP_REFERER', '')
+        params = build_ga_params(
+            request, account, path=path, referer=referer, title=title)
+        response = set_cookie(params, response)
+
+        def calculate_age(dob):
+            dob = datetime.datetime.strptime(dob, '%Y-%m-%d').date()
+            today = get_today()
+            print today, dob
```
Comment (codiebeulaine) [inline iogt/middleware.py]: stray print

**Comment by codiebeulaine** [comment]
I'm wondering if perhaps this should be in molo core to benefit all the projects and not specifically iogt?

File: iogt/middleware.py (hunk 1 above)
Comment (erikh360) [reply iogt/middleware.py]: Ah thanks!

**Comment by erikh360** [comment]
@codiebeulaine We decided to do this here because IoGT is still using a old molo version.

**Comment by codiebeulaine** [comment]
This looks good, I realise iogt is probably not on the latest version of molo, 👍 wtih

**Pull request merged**